examples/shopping_ads/add_performance_max_retail_campaign.py

```diff
@@ -61,16 +61,12 @@
 
 # There are also entities that will be created in the same request but do not
 # need to be fixed temporary IDs because they are referenced only once.
-next_temp_id = int(_ASSET_GROUP_TEMPORARY_ID) - 1
+_NEXT_TEMP_ID = int(_ASSET_GROUP_TEMPORARY_ID) - 1
```

Comment:
good point, looking again at [PEP8](https://peps.python.org/pep-0008/#global-variable-names) this should probably be `_next_temp_id` because it is intended to be a non-public global, and lowercase since it isn't a constant. I'll update the PR.